### PR TITLE
Add: Explanation of calculating t=0.22

### PR DESCRIPTION
**Pull request by nc6**
Calculation due to Yun Lu

### CODE REVIEW

**Comment by dnadales** [comment]
It seems that this should close https://github.com/input-output-hk/cardano-chain/issues/274. Do we want to add this as the PR comment?

**Comment by nc6** [comment]
I've connected the issue

File: specs/chain/latex/blockchain-spec.tex [hunk 1]
```diff
@@ -1039,6 +1043,50 @@ \section{Blockchain Extension}
...
+any one of these windows is bounded above by the sum of the likelihoods for each
+window. We may thus consider that the probability of a given stakeholder
+violating the threshold in an epoch to be bounded by $(n-K+1)\cdot P(X > t*K)$.
+Appealing to Boole's inequality again, we may multiply this by the number of
+epochs and the number of stakeholders to give a bound for the likelihood of
+generating an invalid chain.
+
+Figure \ref{fig:calculating-t} gives the bound on the likelihood of threshold
+violation for $t$ in our plausible range: from this we can see that the
+likelihood decreases to a negligible level around 0.21, and so we choose the
```
Comment (dnadales) [inline specs/chain/latex/blockchain-spec.tex]:
```suggestion
likelihood decreases to a negligible level around $0.21$, and so we choose the
```

**Review by dnadales — APPROVED** [review]

**Pull request merged**